suite-native/atoms/src/Pictogram.tsx

```diff
@@ -99,7 +99,7 @@ export const Pictogram = ({
                     <Icon name={icon} color={iconColor} customSize={ICON_SIZE} />
                 </Box>
             </Box>
-            {title && (
+            {(title || subtitle) && (
```

Comment:
nit: What about moving the `VStack` above the condition and then separately conditionally render title as well as subtitle?

Reply:
9860215781

Reply:
9860215781